integration_tests/config.py

```diff
@@ -12,7 +12,6 @@
                 "PRIMER_DETECTED",
                 "PRIMER_NUMREADS",
                 "PRIMER_UNIQUE_NUMREADS",
-                "PRIMER_AMBIGUOUS_NUMREADS",
```

Comment:
Due to the new search of primers using a window of bases, we can no longer use a jolly character, which was at the core for counting ambiguous primers. Therefore this count is now removed.